Library/Homebrew/cli_parser.rb

```diff
@@ -16,11 +23,13 @@ def initialize(&block)
 
       def switch(*names, description: nil, env: nil)
         description = option_to_description(*names) if description.nil?
-        names, env = common_switch(*names) if names.first.is_a?(Symbol)
+        global_switch = names.first.is_a?(Symbol)
+        names, env = common_switch(*names) if global_switch
         @parser.on(*names, description) do
-          enable_switch(*names)
+          enable_switch(*names, global_switch)
         end
-        enable_switch(*names) if !env.nil? && !ENV["HOMEBREW_#{env.to_s.upcase}"].nil?
+        enable_switch(*names, global_switch) if !env.nil? &&
```

Comment:
I'd use an `if ... end` here.

Reply:
This leads to Rubocop error  : `Use a guard clause instead of wrapping the code inside a conditional expression.`